database/migrations/2021_05_31_141139_create_stores_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateStoresTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('stores', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->integer('category_id');
+            $table->foreign('category_id')->references('id')->on('categories');
+            $table->integer('prefecture_id');
+            $table->foreign('prefecture_id')->references('id')->on('prefectures');
+            $table->string('address');
+            $table->string('phone_number');
+            $table->string('opening_hour');
+            $table->string('closing_day');
+            $table->string('price');
+            $table->string('store_image');
+            $table->string('recommend_product');
+            $table->string('recommend_image');
+            $table->decimal('latitude');
+            $table->decimal('longitude');
```

Comment:
桁数（長さ）の指定をしておきましょっか！
```suggestion
            $table->decimal('latitude', 12, 9);
            $table->decimal('longitude', 12, 9);
```

Reply:
laravelのドキュメントを確認して、合計桁数と小数点以下の桁数と確認しました。
この設定するってことで合っていますか？